test/unit/exception_handling_test.rb

```diff
@@ -360,6 +360,26 @@ def log_error_callback(data, ex)
       end
     end
 
+    context "Honeybadger integration" do
+      setup do
+        class ::Honeybadger
+          def self.notify(exception)
+            "Honeybadger notify."
+          end
+        end
+      end
+
+      should "invoke send_exception_to_honeybadger when log_error is executed" do
+        ExceptionHandling.expects(:send_exception_to_honeybadger).times(1)
+        ExceptionHandling.log_error(exception_1)
+      end
+
+      should "send exception to Honeybadger when send_exception_to_honeybadger is executed" do
```

Comment:
Can you add a test to make sure Honeybadger.notify is not invoked when we call log_error but Honeybadger is not defined?